spring-cloud-kubernetes-integration-tests/spring-cloud-kubernetes-client-config-it/src/test/java/org/springframework/cloud/kubernetes/client/config/it/ConfigMapAndSecretIT.java

```diff
@@ -78,17 +74,35 @@
 
 	private static K8SUtils k8SUtils;
 
+	private static final K3sContainer K3S = new K3sContainer(DockerImageName.parse("rancher/k3s:v1.21.10-k3s1"))
```

Comment:
- start an instance of `K3sContainer` with image `rancher/k3s:v1.21.10-k3s1` as base
- bind `/tmp/images` volume. before when we invoke this piece of code, `docker save ....` already happened, so the image is there
- `withCommand("server")` so that we get `traefik` installed
- `.withReuse(true)` to speed-up the process (we also need `echo "testcontainers.reuse.enable=true" > ~/.testcontainers.properties` for that to work)